apps/components/src/app/toolbar/toolbar.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { NgpToolbar } from 'ng-primitives/toolbar';
+
+@Component({
+  selector: '[app-toolbar]',
```

Comment:
I think we can just use an element selector, `div` adds no special semantics compared to a div so we can just use the element selector here.

```suggestion
  selector: 'app-toolbar',
```